test/EVMHost.cpp

```diff
@@ -249,13 +249,33 @@ evmc::result EVMHost::call(evmc_message const& _message) noexcept
 
 	if (message.kind == EVMC_CREATE)
 	{
-		// TODO this is not the right formula
 		// TODO is the nonce incremented on failure, too?
+		sender.nonce++;
+
+		auto encodeRlpInteger = [](int value) -> bytes {
+			if (value == 0) {
+				return bytes(1, 128);
```

Comment:
```suggestion
				return bytes{128};
```
You can create it like this.